src/components/editor/RDFModal.jsx

```diff
@@ -1,52 +1,67 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
-import { bindActionCreators } from 'redux'
-import { connect } from 'react-redux'
-import Modal from 'react-bootstrap/lib/Modal'
-import Row from 'react-bootstrap/lib/Row'
-import Col from 'react-bootstrap/lib/Col'
+import { hideModal } from 'actions/index'
+import { connect, useDispatch } from 'react-redux'
 import PropTypes from 'prop-types'
 import GraphBuilder from 'GraphBuilder'
-import { showRdfPreview } from 'actions/index'
+import ModalWrapper from './ModalWrapper'
 import SaveAndPublishButton from './SaveAndPublishButton'
 
 const RDFModal = (props) => {
-  if (!props.show) {
-    return null
+  const dispatch = useDispatch()
+
+  const classes = ['modal', 'fade']
```

Comment:
Probably want to factor this into a hook. Can be a separate ticket.

Reply:
Now ticketed in #1553